src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -189,7 +189,7 @@ private function initializeSubscribers(): void
                 $listener = $this->container->get($listener);
             }
             // throw new \InvalidArgumentException(sprintf('Using Doctrine subscriber "%s" is not allowed, declare it as a listener instead.', \is_object($listener) ? $listener::class : $listener));
-            trigger_deprecation('symfony/doctrine-bridge', '6.3', 'Using Doctrine subscribers as services is deprecated, declare listeners instead');
+            trigger_deprecation('symfony/doctrine-bridge', '6.3', 'Implementing "%s" in "%s" is deprecated, declare it as a listener instead.', EventSubscriber::class, \is_object($listener) ? get_debug_type($listener) : $listener);
```

Comment:
we didn't deprecate implementing the interface, but we deprecated relying on it to register listeners
```suggestion
            trigger_deprecation('symfony/doctrine-bridge', '6.3', 'Registering "%s" as a Doctrine subscriber is deprecated. Register it as a listener with the #[AsDoctrineListener] attribute instead.', EventSubscriber::class, \is_object($listener) ? get_debug_type($listener) : $listener);
```

Reply:
Applied: I removed `EventSubscriber::class` as there is only one placholder in your suggestion